tests/lambda_endpoints/test_query_api.py

```diff
@@ -59,18 +59,18 @@ def expected_response():
     expected_response = json.dumps(
         str(
             [
-                (
-                    1,
-                    "test/file/path/imap_hit_l0_raw_20251107_20251108_v02-01.pkts",
-                    "hit",
-                    "l0",
-                    "raw",
-                    datetime.datetime(2025, 11, 7, 0, 0),
-                    datetime.datetime(2025, 11, 8, 0, 0),
-                    "v02-01",
-                    "pkts",
-                    1,
-                )
+                {
+                    "id": 1,
+                    "file_path": "test/file/path/imap_hit_l0_raw_20251107_20251108_v02-01.pkts",
```

Comment:
If it is just this link causing issues you can add an inline noqa as well if you find that more helpful so it doesn't ignore other lines within the file.
```suggestion
                    "file_path": "test/file/path/imap_hit_l0_raw_20251107_20251108_v02-01.pkts",  # noqa: E501
```

Could you add a quick test that asserts you can `json.loads(response["body"])` as well?

Reply:
Thanks! I always forget about that handy `noqa` thing.